recipes/libressl/2.9.2/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+if(EXISTS "${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+else()
+  include(conanbuildinfo.cmake)
+endif()
+conan_basic_setup()
+
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Why do you need C++11 if this project is pure C?

Reply:
ups